Cargo.toml

```diff
@@ -53,7 +53,7 @@ name = "key"
 
 [features]
 default = ["std", "pem"]
-hazmat = ["signature/hazmat-preview"]
+hazmat = []
```

Comment:
Can probably remove the feature gating here entirely

Reply:
Well, for me the goal was to prevent hazmat interfaces from being available by default. I can drop it of course.